### DISCUSSION TITLE
Reinventing the Remote Control wheel ??

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#426] [General]
At the risk of stating the obvious or teaching granny to suck eggs.....

I came across this when I was researching Slingbox alternatives a couple of weeks ago. 

I found it here [JP1 Remotes :: View topic - Slingbox news (hifi-remote.com)](http://www.hifi-remote.com/forums/viewtopic.php?p=145528#145528)  See its contents below
__________________________________________________________________________________________________________________________
"As Sling's servers go into Sunset. Here's a copy of the latest firmware files for models up to ProHD as a backup archive. These can be uploaded to your Slingbox using the older SlingPlayer EU-1.5.1.343 PC or Mac 1.0.10 Setup Assistants. The only need for these old players is for setup and/or reconfiguration of boxes. Once setup and/or reconfigured it's best (has better quality playback) to use the Ver 2 SlingPlayer for PC (prior to 2.0.3) for direct IP video streaming or Slinger (https://github.com/GerryDazoo/Slinger)."

https://www.mediafire.com/file/8anu6bc35iwp2um/firmwaredb.zip/file

***** Also this is the entire and complete archive of Sling's remote control files (including skins for players and BIN files).

https://www.mediafire.com/file/ti7a9jwtrjtaop7/remotedb.zip/file

Within this zip is the IR-SBAV folder of the various model Slingbox firmware bins; that can be uploaded to you box as a custom remote. Inside these folders are CSV files with information on which remote BIN is applicable to each remote model. For info on how to do this see 

https://slingbox.fandom.com/wiki/How_to_upload_custom_remote_.BIN_files_into_your_Slingbox
_________________________________________________________________________________________________________________

I've copied it in its entirety but what it has links to is hundreds of Remote Controls for Slingboxes. Don't ask me what the various components are for - I've no idea but some of you guys will almost certainly know.

As an example, here is a Virginmedia Tivo remote that is almost identical to my physical device. The file name for it is D3894566-7C68-429b-8C7B-E64C46768790_Big.png. There is also a "SMALL" version of it 

![image](https://user-images.githubusercontent.com/117101460/206217037-c5f5b93b-b6f0-463c-892c-532a344d19d6.png)

So I'm not sure if any of this gives a leg up to those creating newer remote controls to work with Slinger.

Cheers,

Steve

**Comment by DoogieHowitzerWMD** [comment]
See here for discussion of the .bin files #415 

See here for fancy things people have been able to do with remote files that work with Slinger (some were able to incorporate the images of the remotes) #211 #224
